content/events/16-17/latex.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Latex les
+image: //puu.sh/rFa9H/8d568d4a54.png
+description: Introductie tot LaTeX
+created_at: 11-10-2016
+time: 25-10-2016 19:00
+location: Auditorium D, Plateau
+locationlink: plateau, gent
+---
+
+
+Heb je even genoeg van Word? Of wil je gewoon de alternatieven leren kennen?
+Kom eens langs op onze LaTeX les!
+
+Latex laat je toe om grote documenten te schrijven zoals thesissen, reports en papers met veel meer controle dan in word.
```

Comment:
Latex -> LaTeX
+ beter:
LaTeX laat je toe grote documenten zoals thesissen, rapporten en papers te schrijven met veel meer controle dan in Word.